pyproject.toml

```diff
@@ -50,7 +50,7 @@ Markdown = "^3.1"
 [tool.poetry.dev-dependencies]
 beautifulsoup4 = "^4.8"
 coverage = "^6.2"
-flake8 = "^3.7"
+flake8 = "^7.0"
```

Comment:
you need to run `poetry lock --no-update` locally. This should update the lock file for the flake8 change, leaving everything else as is

Reply:
Poetry keeps looking for Python 3.10 which was removed from my system long ago.